Sources/Prometheus/Utils.swift

```diff
@@ -1,28 +1,29 @@
 import Foundation
 
-/// Prometheus global dispatch queue
-internal let prometheusQueue = DispatchQueue(label: "prometheus.internal", attributes: .concurrent)
-
```

Comment:
glad that we don't need to use dispatch for the lib <3

Reply:
Same, hahahaha